web/client-api/src/main/java/io/deephaven/web/client/api/JsColumnStatistics.java

```diff
@@ -36,7 +36,7 @@ public enum StatType {
         /**
          * The number of unique values in the column.
          */
-        UNIQUE_VALUES("UNIQUE VALUES", "long"),
+        UNIQUE_VALUES("UNIQUE VALUES", "int"),
```

Comment:
Just making sure this is intentional to drop from a long to an int. We should probably check that the client puts a much smaller limit on this (trying to show the count of 10000 unique values is going to be a bad time in the web)

Reply:
Confirmed this has a visual limit in the UI. No idea where it chose the order from though or if the API is limiting data sent across for this (this is just a 500 column table w/ string value of i for the column)

![image](https://github.com/user-attachments/assets/7a6c0b89-ad7d-4780-b3ba-98959e48f0b1)

Reply:
A couple of things are happening here:

Yes, that "unique value _count_" is an int, not a long - the server won't end a number bigger than 2 billion, as that would require it to have actually tracked 2 billion+ unique values, and our collection types won't differentiate past 2^31 values.

Separately, there is a "unique value list with counts", which will be no larger than the "unique value count" above. That describes the maximum number of values to display in the list that you see in the UI. This will find the top N most common values in the column sorted by their counts. By default, N is 20, though the client can set this as high as 200. Separately, there is a limitation that if more than 1 million unique values are seen while scanning the column, the counts won't be collected, we'll just count how many unique values there are.

All three of those values are server configuration properties:
ColumnStatistics.defaultUniqueLimit=20
ColumnStatistics.maxUniqueLimit=200
ColumnStatistics.maxUniqueToCollect=1000000

Presently, we don't expose the limit through the JS API, reflecting how DHE handles this.